### PR TITLE
Feature/map component

### PR DESCRIPTION
**Pull request by MLucky518**

### CODE REVIEW

File: src/components/pages/Profile/Profile.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const Profile = props => {
   useEffect(() => {
     // Check current user type to see if groomer or not
 
-    if (props.currentUser.user_type === 'groomer') {
+    if (props.currentUser.is_groomer) {
```
Comment (wilcoxva) [inline src/components/pages/Profile/Profile.js:13]: thanks for fixing this!

File: src/components/pages/ProfileList/profileList.scss [hunk 2]
```diff
@@ -1,20 +1,29 @@
...
+@import url(https://fonts.googleapis.com/css?family=Amatic+SC:700);
+
 .profile-container {
-    background-image: url("https://i.pinimg.com/originals/a3/00/48/a300488ab0270733e98fec22965cf9ed.jpg");
-    height:150vh;
-    background-size: cover;
-    background-repeat: no-repeat;
-    background-attachment: fixed;
-    background-position: center;
+  background-image: url('https://i.pinimg.com/originals/a3/00/48/a300488ab0270733e98fec22965cf9ed.jpg');
```
Comment (wilcoxva) [inline src/components/pages/ProfileList/profileList.scss:11]: cute img!

File: src/components/Map/MapBox.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  );
+
+  const handleGeocoderViewportChange = useCallback(newViewport => {
+    const geocoderDefaultOverrides = { transitionDuration: 1000 };
+
+    return handleViewportChange({
+      ...newViewport,
+      ...geocoderDefaultOverrides,
+    });
+  }, []);
```
Comment (tlewandowski18) [inline src/components/Map/MapBox.js:39]: Code looks really clean and readable

File: src/components/pages/Home/HomeContainer.js [hunk 4]
```diff
@@ -26,8 +26,8 @@ function HomeContainer({ LoadingComponent }) {
         isSubscribed = false;
         return setUserInfo(null);
       });
-    const id = userInfo.id;
-    getUserProfileData(authState, id);
+
+    getUserProfileData(authState);
```
Comment (tlewandowski18) [inline src/components/pages/Home/HomeContainer.js:30]: I think we actually changed this to include the id parameter

File: src/state/reducers/appReducer.js [hunk 5]
```diff
@@ -11,12 +11,14 @@ export const appReducer = (state = initialState, action) => {
       console.log(action.payload);
       return {
         ...state,
-        profiles: state.profiles.append(action.payload),
+        profiles: action.payload,
```
Comment (tlewandowski18) [inline src/state/reducers/appReducer.js:14]: Good pickup here

**Review by tlewandowski18 — APPROVED** [review]

**Pull request merged**